packages/gsound.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/autotools'
 
-class Gsound < Package
+class Gsound < Autotools
   description 'GSound is a small library for playing system sounds.'
   homepage 'https://wiki.gnome.org/Projects/GSound'
   version '1.0.2'
```

Comment:
Update to [1.0.3](https://download.gnome.org/sources/gsound/1.0/gsound-1.0.3.tar.xz)?